src/main-enhancer.js

```diff
@@ -20,7 +20,7 @@ const defaultParams = {
   postDispatchCallback: () => null,
   preDispatchCallback: () => null,
   dispatchProxy: null,
-  actionFilter: () => true,
+  actionFilter: () => true
```

Comment:
```suggestion
  actionFilter: () => true,
```
Could this be reverted?  I like to keep a trailing comma so that its easier to move lines around + adding a new line at the bottom doesn't require a change to the line above it in the diff